src/OpenTelemetry.Api/OpenTelemetry.Api.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
```

Comment:
What exactly does "descending order" mean here? Is this still in "descending order"?

Reply:
Thanks for pointing this out. This seems to originate here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/2015#discussion_r622391733

And that in turn says that VS Code only uses the first TFM in the list. So this would change that selection to .NET 6.
Personally I would argue that's a good thing. And it's also consistent with `OpenTelemetry.csproj` which also starts with `net6.0`.